dataminer/Troubleshooting/Known_issues/KI_Trend_flatline_different_timezones.md

```diff
@@ -0,0 +1,33 @@
+﻿---
+uid: KI_Trend_flatline_different_timezones
+---
+
+# Trend graph flatline when client and DMA use different timezones
+
+## Affected versions
+
+This issue occurs in DataMiner 10.5.8 and later.
```

Comment:
Does the issue also occur on the Main Release track? If yes, we'll have to add the MR version where it starts to occur as well.